### ISSUE TITLE
simple_bucket is missing some lifecycle rules options

### ISSUE THREAD

**Issue by jawnsy** [terraform-google-modules/terraform-google-cloud-storage#144]
### TL;DR

The main module supports several lifecycle rules that the simple bucket submodule does not

### Expected behavior

I expected both to support the same lifecycle rules

### Observed behavior

The simple_bucket submodule is missing several lifecycle rules.

The primary module has the "days_since_noncurrent_time" lifecycle rule (there are 9 rules in total):

https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/763cf841b255a8dbd627f7334201e8773aee1769/main.tf#L107-L126

However, the simple_bucket submodule does not have that rule (there are 5 rules in total):

https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/763cf841b255a8dbd627f7334201e8773aee1769/modules/simple_bucket/main.tf#L55-L70

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
$ terragrunt version
Terraform v1.0.11
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v3.89.0
+ provider registry.terraform.io/hashicorp/google-beta v3.89.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/integrations/github v4.18.0
```

**Comment by morgante** [comment]
This is not a bug. it is a feature request. Recategorizing as an *enhancement*.

We are unlikely to fix this soon. You are welcome to file a pull request and we will review it.
